diff --git a/third_party/llvm/workspace.bzl b/third_party/llvm/workspace.bzl
index 622428b..b1675f1 100644
--- a/third_party/llvm/workspace.bzl
+++ b/third_party/llvm/workspace.bzl
@@ -4,8 +4,8 @@ load("//third_party:repo.bzl", "tf_http_archive")
 
 def repo(name):
     """Imports LLVM."""
-    LLVM_COMMIT = "d5ce81dc8143eed18a7342093b991a63b025e2d9"
-    LLVM_SHA256 = "07efbd552b4539a0b52201cfe73767a4265720789facdfa0d73bfeeb78ea19d3"
+    LLVM_COMMIT = "32de3b9ef9e7e8debc14416e968456ca13b48bea"
+    LLVM_SHA256 = "e048b05e1fb9366e224ea3c06f8473714114039bfad00e81db4ecb6409f23efa"
 
     tf_http_archive(
         name = name,
